lib/completions/endpoints/hugging_face.rb

```diff
@@ -56,15 +58,15 @@ def extract_completion_from(response_raw)
 
             parsed.dig(:token, :text).to_s
           else
-            parsed[:generated_text].to_s
+            parsed[0][:generated_text].to_s
```

Comment:
Consequence of the move to the header for streaming is this slight response format change